ImperatorToCk2/src/main/java/ImperatorToCK2/Processing.java

```diff
@@ -1296,4 +1297,73 @@ public static String calcHead (ArrayList<String[]> impCharInfoList, String dynas
 
     }
 
+    public static String deriveRgbFromHsv (String color) //Converts HSV to RGB, adapted from commonItems
```

Comment:
It might be worth copying the whole Color class. Then you can make it that you just pass a string to the Color to build it instead of having the same code to figure it out copied several places.